cloud-db/queries/android.md

```diff
@@ -102,6 +102,56 @@ Besides the operations shown above, the following list out all operations suppor
 
 ## Relational Queries
 
+This example shows how to query all notes (`Note` record) who has an `account` field reference to a user record. In this example, we will query all notes where `account` equals to the current user.
+
+```java
+Record currentUser = Container.defaultContainer(this).getAuth().getCurrentUser(); // Get the current user
+Query noteQuery = new Query("Note").transientInclude("account")
```

Comment:
First example should ignore transientInclude? Or leave it as a comment to say sth like // .transientInclude("account") // let's leave eager loading alone now, will explain in next section.